strawberry/permission.py

```diff
@@ -74,7 +87,7 @@ def on_unauthorized(self) -> None:
         raise error
 
     @property
-    def schema_directive(self) -> object:
+    def schema_directive(self) -> List[object]:
```

Comment:
This is a Breaking change, can we instead introduce some new directive logic for @And and @Or? Let's discuss this in the PR